infrastructure/main.tf

```diff
@@ -118,6 +118,10 @@ data "vault_generic_secret" "puiWebapp" {
   path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/pui-webapp"
 }
 
+data "vault_generic_secret" "bulkScanUpload" {
+  path = "secret/${var.vault_section}/ccidam/service-auth-provider/api/microservice-keys/bulk-scan-upload"
```

Comment:
maybe 'processor' instead of 'upload', for consistency?
(https://github.com/hmcts/bulk-scan-processor)